ci/do_ci.sh

```diff
@@ -9,29 +9,31 @@ echo "building using ${NUM_CPUS} CPUs"
 
 if [[ "$1" == "bazel.debug" ]]; then
   echo "bazel debug build with tests..."
-  ${BAZEL} build ${BAZEL_BUILD_OPTIONS} //source/exe:envoy-static
-  ${BAZEL} test ${BAZEL_BUILD_OPTIONS} --test_output=all \
-    --cache_test_results=no //test/...
+  cd "${ENVOY_CONSUMER_SRCDIR}"
```

Comment:
Yup makes sense. As long as we do at least one build that doesn't use consumer I think that's fine.